scripts/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+# Start chat service in the background
```

Comment:
Why are you starting it in the background ?
I assumed we would want ejabberd to log to stdout so we see its logs with `kubectl logs` and also so they would automatically get shipped to logz.io

Reply:
Logs still get sent to stdout.

It's weird, man.

Reply:
Discussed offline, this still logs to `stdout` :+1: